buildpacks/jvm-function-invoker/src/detect.rs

```diff
@@ -0,0 +1,62 @@
+use std::path::Path;
+
+use libcnb::data::build_plan::BuildPlanBuilder;
+use libcnb::{read_toml_file, DetectContext, DetectOutcome, Error, GenericPlatform};
+
+use crate::error::JvmFunctionInvokerBuildpackError;
+use crate::JvmFunctionInvokerBuildpackMetadata;
+use libherokubuildpack::toml_select_value;
+use toml::Value;
```

Comment:
Local and crate `use` statements are interleaved here - should they be grouped separately? (Ideally rustfmt would handle this, but https://github.com/rust-lang/rustfmt/issues/2979)

Reply:
Personally, I don't care a lot about import order. Ideally, we would not add our own rules on top of `rustfmt` so we can reap the benefits of an opinionated formatter: neither the author nor reviewers need to waste energy and time to enforce/check formatting.

To be clear: I'm not opposed to change this here, but I also think it isn't important at all.